src/hera/workflows/v5/script.py

```diff
@@ -23,21 +23,21 @@
 
 class Script(
     _IOMixin,
-    _DAGTaskMixin,
     _ContainerMixin,
     _EnvMixin,
     _TemplateMixin,
     _ResourceMixin,
     _VolumeMountMixin,
     _SubNodeMixin,
 ):
-    name: str
+    container_name: Optional[str] = None
     args: Optional[List[str]] = None
     command: Optional[List[str]] = None
     lifecycle: Optional[Lifecycle] = None
     security_context: Optional[SecurityContext] = None
     source: Optional[Union[Callable, str]] = None
     working_dir: Optional[str] = None
+    add_cwd_to_sys_path: bool = True
```

Comment:
Added this option so that users have control over whether they want to modify syspath or not. This leads to cleaner output scripts.